classes/mender-install.bbclass

```diff
@@ -13,11 +6,4 @@ VIRTUAL-RUNTIME_init_manager = "systemd"
 DISTRO_FEATURES_BACKFILL_CONSIDERED = "sysvinit"
 VIRTUAL-RUNTIME_initscripts = ""
 
-
-#Make sure we are creating sdimg with all needed partitioning.
-IMAGE_CLASSES += "sdimg"
-IMAGE_FSTYPES += "sdimg"
-
-IMAGE_INSTALL_append = " u-boot-fw-utils mender ca-certificates"
-
-
+IMAGE_INSTALL_append = " mender ca-certificates"
```

Comment:
Try bbappending `packagegroup-core-boot`. Mender kind of interacts with the boot process, so it might fit wit the other packages in `packagegroup-core-boot`.

It's also good if we document explicitly that `mender` package will be added to `core-boot` group, so that people are not surprised.

BTW. I think that we don't really have a dependency on `ca-certificates`. First we're shipping our own server certificate. Secondly, as far as I know, Go stdlib does not look for certificates in `${sysconfdir}/ca-certificates` or `${datadir}/ca-certificates`